tests/js/server/aql/aql-persistent-index.js

```diff
@@ -1552,8 +1596,137 @@ function PersistentIndexOverlappingSuite () {
   };
 }
 
+function PersistentIndexSelectivityTestSuite () {
+  return {
+    setUp: function () {
+      internal.db._drop(cn);
+      let c = internal.db._create(cn);
+      let docs = [];
+      for (let i = 0; i < 10; ++i) {
+        docs.push({value: i});
+      }
+      c.insert(docs);
+    },
+
+    tearDown: function () {
+      internal.db._drop(cn);
+    },
+
+    testUniqueSimple: function () {
+      testandValidate(query1, true, false, false, false, false, 1);
+    },
+
+    testUniqueSimple2: function () {
+      testandValidate(query2, true, false, false, false, false, 1);
+    },
+
+    testNonUniqueSimple: function () {
+      testandValidate(query1, false, false, false, false, false, 1);
+    },
+
+    testNonUniqueSimple2: function () {
+      testandValidate(query2, false, false, false, false, false, 1);
+    },
+
+    testUniqueSimpleManyDocs: function () {
+      if (isCluster) {
```

Comment:
the `isCluster` checks here and following are not necessary, because the entire suite is only run if `!isCluster`.